### DISCUSSION TITLE
Cannot connect Windows to Windows

### DISCUSSION THREAD

**Discussion by testergrooves** [apohl79/audiogridder#1285] [Help]
Been at it a few hours. Tried everything I could possibly think of. All I get is "not connected." I added the IP, I added firewall exceptions, etc nothing makes any difference. They're on the same WiFi. Server is a Dell G5, client is a Surface Pro X. Is it because the Surface is ARM?

**Comment by apohl79** [comment]
It is the firewall most of the time. Try to disable it for testing and see if that works.
